eva_cttv_pipeline/consequence_type.py

```diff
@@ -78,104 +78,106 @@ def process_consequence_type_file(snp_2_gene_file):
     return consequence_type_dict
 
 
-class ConsequenceType(object):
-    class SoTerm(object):
-
-        so_accession_name_dict = {}
```

Comment:
This dictionary could be initialized in a more Pythonic way, like:

`so_accession_name_dict = {
 'transcript_ablation' : 1893,
 'splice_donor_variant' : 1575,
 ...
}`

Still multi-line, GitHub reviews only accept one-liners.